.eslintrc.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     // We enable the rule in src/.eslintrc.js since that's the only place we
     // want to disallow importing extraneous dependencies.
     "import/no-extraneous-dependencies": "off",
-    "prefer-destructuring": "off"
+    "prefer-destructuring": "off",
+    "import/prefer-default-export": "off"
```

Comment:
Whenever we start a constants file where we know there may eventually be multiple constant values (like `httpHeaderNames.js`), we'd run into this eslint error and end up ignore it. I decided to disable the rule and expect us to just be judicious about when we use a default export vs named exports. If you disagree with this change, please let me know.